proxy/proxy.go

```diff
@@ -1489,7 +1489,7 @@ func (p *Proxy) setCommonSpanInfo(u *url.URL, r *http.Request, s ot.Span) {
 		setTag(s, HTTPPathTag, u.Path).
```

Comment:
I guess this was missing

Reply:
I tested that if you parse as we do with url.Parse all attributes are encoded correctly (path,query,hostname), the same for NewRequest(), so I don't know. I will try to remove the setTag encoder to test another assumption